planet/clients/data.py

```diff
@@ -336,7 +337,7 @@ async def get_stats(self,
             interval: The size of the histogram date buckets.
 
         Returns:
-            Returns a date bucketed histogram of items matching the filter.
+            A full JSON description of the returned statistics result.
```

Comment:
What is the reasoning behind this change? The prior description seemed more, well, descriptive.

Reply:
@jreiberkyle I added the word "histogram" to this for context, but this was changed here to clarify that it is returned in JSON form. Generally I understand "histogram" to indicate a graphical/chart representation so this is just for clarification that it is a description of one, not an actual histogram.